libs/designer-ui/src/lib/floatingactionmenu/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { KeyCodes } from '@fluentui/react';
+import { useBoolean } from '@fluentui/react-hooks';
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+export enum DynamicallyAddedParameterType {
+  Text = 1,
+  File,
+  Email,
+  Boolean,
+  Number,
+  Date,
+}
+
+export interface FloatingActionMenuItem {
+  id: number;
+  icon: string;
+  label: string;
+}
+
+export interface FloatingActionMenuProps {
+  collapsedTitle: string;
+  expandable: boolean;
+  expandedTitle: string;
+  menuItems: FloatingActionMenuItem[];
+  onMenuItemSelected(item: FloatingActionMenuItem): void;
+}
+
+export const FloatingActionMenu = (props: FloatingActionMenuProps): JSX.Element => {
+  const [expanded, { toggle: toggleExpanded }] = useBoolean(false);
+
+  const toggleExpandedOnKeyDown = (e: React.KeyboardEvent<HTMLElement>) => {
+    const { keyCode } = e;
+
+    if (keyCode === KeyCodes.enter || keyCode === KeyCodes.space) {
+      e.preventDefault();
+      e.stopPropagation();
+      toggleExpanded();
+    }
+  };
+
+  const renderMenuButton = (): JSX.Element => {
+    return (
+      <div role="button" className="msla-floating-action-menu" onClick={toggleExpanded} onKeyDown={toggleExpandedOnKeyDown} tabIndex={0}>
+        <span className="msla-floating-action-menu-plus-icon">{'+'}</span>
```

Comment:
should there be double quotes around the +? `"+"`?

Reply:
no single quotes unless the string has a single quote inside of it. Prettier will always enforce this.

Reply:
Prettier puts these to single quotes, going with that